<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>522</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bike Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/heckview.jpg</normaloff>images/heckview.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>481</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_analyis">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="tiltCheckbox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Tilt angle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="steeringCheckbox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Steering angle</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="speedCheckbox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Speed profile</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="obstaclesCheckbox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ostacles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ostacles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_testing">
     <attribute name="title">
      <string>Testing</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>101</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_riding">
     <attribute name="title">
      <string>Riding</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="BatteryLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Battery</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="batteryState">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="value">
        <number>75</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
      <widget class="QLabel" name="batteryImage">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/348px-Angular_lightningbolt.svg.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="emergencyStopButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/Stop.png</normaloff>images/Stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="incSpeedButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>22</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/500px-Aiga_uparrow_inv.svg.png</normaloff>images/500px-Aiga_uparrow_inv.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="decSpeedButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/500px-Aiga_downarrow_inv.svg.png</normaloff>images/500px-Aiga_downarrow_inv.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="turnRightButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/500px-Aiga_rightarrow_inv.svg.png</normaloff>images/500px-Aiga_rightarrow_inv.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="turnLeftButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/500px-Aiga_leftarrow_inv.svg.png</normaloff>images/500px-Aiga_leftarrow_inv.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="currentSpeed">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>2</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="kmhLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Km/h</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="tilt">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="tiltImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>141</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/heckview.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="steeringImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>141</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/topview.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="obstaclesImage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>141</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/backyard-obstacle-course-map.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="msgList">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>93</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="tiltButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>tilt</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="steeringButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>steering</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="obstaclesButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>obstacles</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="videoButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>video</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionSet_IP"/>
    <addaction name="actionLock_settings"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <action name="actionSet_IP">
   <property name="text">
    <string>Set IP</string>
   </property>
  </action>
  <action name="actionLock_settings">
   <property name="text">
    <string>Lock settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>emergencyStopButton</tabstop>
  <tabstop>msgList</tabstop>
  <tabstop>incSpeedButton</tabstop>
  <tabstop>turnLeftButton</tabstop>
  <tabstop>turnRightButton</tabstop>
  <tabstop>decSpeedButton</tabstop>
  <tabstop>tiltButton</tabstop>
  <tabstop>steeringButton</tabstop>
  <tabstop>obstaclesButton</tabstop>
  <tabstop>videoButton</tabstop>
  <tabstop>speedCheckbox</tabstop>
  <tabstop>tiltCheckbox</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>steeringCheckbox</tabstop>
  <tabstop>obstaclesCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tiltButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tiltImage</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>steeringButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>steeringImage</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>obstaclesButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>obstaclesImage</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
